scripts/configs/llm_pddl.yaml

```diff
@@ -0,0 +1,120 @@
+      pddl_blocks_procedural_train_min_num_blocks_goal: 6
+      pddl_blocks_procedural_train_max_num_blocks_goal: 7
+      pddl_blocks_procedural_test_min_num_blocks: 9
+      pddl_blocks_procedural_test_max_num_blocks: 10
+      pddl_blocks_procedural_test_min_num_blocks_goal: 6
+      pddl_blocks_procedural_test_max_num_blocks_goal: 9
+  easy_delivery:
+    NAME: "pddl_delivery_procedural_tasks"
+    FLAGS:
+      pddl_delivery_procedural_train_min_num_locs: 3
+      pddl_delivery_procedural_train_max_num_locs: 5
+      pddl_delivery_procedural_train_min_want_locs: 1
+      pddl_delivery_procedural_train_max_want_locs: 2
+      pddl_delivery_procedural_test_min_num_locs: 4
+      pddl_delivery_procedural_test_max_num_locs: 6
+      pddl_delivery_procedural_test_min_want_locs: 2
+      pddl_delivery_procedural_test_max_want_locs: 3
+      pddl_delivery_procedural_test_max_extra_newspapers: 1
+  medium_delivery:
+    NAME: "pddl_delivery_procedural_tasks"
+    FLAGS:
+      pddl_delivery_procedural_train_min_num_locs: 4
+      pddl_delivery_procedural_train_max_num_locs: 7
+      pddl_delivery_procedural_train_min_want_locs: 1
+      pddl_delivery_procedural_train_max_want_locs: 3
+      pddl_delivery_procedural_test_min_num_locs: 17
+      pddl_delivery_procedural_test_max_num_locs: 23
+      pddl_delivery_procedural_test_min_want_locs: 11
+      pddl_delivery_procedural_test_max_want_locs: 16
+      pddl_delivery_procedural_test_max_extra_newspapers: 5
+  easy_spanner:
+    NAME: "pddl_spanner_procedural_tasks"
+    FLAGS:
+      pddl_spanner_procedural_train_min_nuts: 1
+      pddl_spanner_procedural_train_max_nuts: 2
+      pddl_spanner_procedural_train_min_extra_spanners: 0
+      pddl_spanner_procedural_train_max_extra_spanners: 2
+      pddl_spanner_procedural_train_min_locs: 1
+      pddl_spanner_procedural_train_max_locs: 3
+      pddl_spanner_procedural_test_min_nuts: 2
+      pddl_spanner_procedural_test_max_nuts: 4
+      pddl_spanner_procedural_test_min_extra_spanners: 0
+      pddl_spanner_procedural_test_max_extra_spanners: 3
+      pddl_spanner_procedural_test_min_locs: 2
+      pddl_spanner_procedural_test_max_locs: 4
+  medium_spanner:
+    NAME: "pddl_spanner_procedural_tasks"
+    FLAGS:
+      pddl_spanner_procedural_train_min_nuts: 1
+      pddl_spanner_procedural_train_max_nuts: 3
+      pddl_spanner_procedural_train_min_extra_spanners: 0
+      pddl_spanner_procedural_train_max_extra_spanners: 2
+      pddl_spanner_procedural_train_min_locs: 2
+      pddl_spanner_procedural_train_max_locs: 4
+      pddl_spanner_procedural_test_min_nuts: 6
+      pddl_spanner_procedural_test_max_nuts: 7
+      pddl_spanner_procedural_test_min_extra_spanners: 0
+      pddl_spanner_procedural_test_max_extra_spanners: 5
+      pddl_spanner_procedural_test_min_locs: 5
+      pddl_spanner_procedural_test_max_locs: 6
+ARGS:
+  - "debug"
+FLAGS:
+  num_train_tasks: 5
+  num_test_tasks: 10
+  strips_learner: "oracle"
```

Comment:
wait...why would LLMs need this?!

Reply:
are you asking why the LLMs need to know operators?

Reply:
(it's because the output of the LLM is converted into operators)